### PR TITLE
Use the stable protobuf definitions

### PR DESCRIPTION
**Pull request by pienkowb**

### CODE REVIEW

**Review by lavabyrd — APPROVED** [review]

File: chain/tendermint/src/protobuf/fig.tendermint.codec.v1.rs [hunk 1]
```diff
@@ -60,11 +58,10 @@ pub struct Commit {
 pub struct CommitSig {
     /// must match those in the enum above
     #[prost(enumeration = "BlockIdFlag", tag = "1")]
-    pub r#type: i32,
-    /// hash 256 20 bytes - this is the first 20 characters of a 32-byte key -
+    pub block_id_flag: i32,
```
Comment (patiee) [inline chain/tendermint/src/protobuf/fig.tendermint.codec.v1.rs:61]: that needs bo reverted

File: chain/tendermint/src/protobuf/fig.tendermint.codec.v1.rs [hunk 2]
```diff
@@ -157,7 +154,7 @@ pub struct EventValidatorSetUpdates {
 pub struct EventVote {
     /// should be present in the enum at the top of this file
     #[prost(enumeration = "SignedMsgType", tag = "1")]
-    pub r#type: i32,
+    pub event_vote_type: i32,
```
Comment (patiee) [inline chain/tendermint/src/protobuf/fig.tendermint.codec.v1.rs:157]: same

File: chain/tendermint/src/runtime/abi.rs [hunk 3]
```diff
@@ -378,7 +378,7 @@ impl ToAscObj<AscCommitSig> for codec::CommitSig {
         heap: &mut H,
     ) -> Result<AscCommitSig, DeterministicHostError> {
         Ok(AscCommitSig {
-            block_id_flag: match self.r#type {
+            block_id_flag: match self.block_id_flag {
```
Comment (patiee) [inline chain/tendermint/src/runtime/abi.rs:381]: same

File: chain/tendermint/src/runtime/abi.rs [hunk 4]
```diff
@@ -571,7 +571,7 @@ impl ToAscObj<AscEventVote> for codec::EventVote {
         heap: &mut H,
     ) -> Result<AscEventVote, DeterministicHostError> {
         Ok(AscEventVote {
-            event_vote_type: match self.r#type {
+            event_vote_type: match self.event_vote_type {
```
Comment (patiee) [inline chain/tendermint/src/runtime/abi.rs:574]: same

**Pull request merged**